real_check/test/both/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+set(real_check_test_both_c_files
```

Comment:
Can we always have a newline after the copyright header?
Easier to read (this way I have to glance twice on it to figure out where the "code" starts and where the comment header ends.

Reply:
Here and in all other files

Reply:
Yes, done in next iteration.